test/ui/presets.test.js

```diff
@@ -0,0 +1,205 @@
+const timeout = process.env.SLOWMO ? 30000 : 100000;
+beforeAll(async () => {
+  await page.goto('index.html', {waitUntil: 'domcontentloaded'});
```

Comment:
aha - ok we're supposed to use this format:

```suggestion
    path = fs.realpathSync('file://../index.html');
    await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
```

https://github.com/publiclab/image-sequencer/blob/1618a97b82480485299cba8c36f85449251494b8/test/ui-2/test/ColorPicker.test.js#L4-L5